src/main/java/org/commcare/formplayer/beans/menus/QueryResponseBean.java

```diff
@@ -63,8 +64,8 @@ public QueryResponseBean(QueryScreen queryScreen, SessionWrapper session) {
                     }
                 }
                 if (indicesForSelectedChoices.size() > 0) {
-                    currentAnswer = String.join(RemoteQuerySessionManager.ANSWER_DELIMITER,
```

Comment:
We would need this to be backward compatible as well to start with to remove the dependency between HQ and Formplayer deploy.